.github/workflows/benchmark-flow.yml

```diff
@@ -124,3 +124,7 @@ jobs:
               --redistimeseries_host ${{ secrets.PERFORMANCE_RTS_HOST }}
               --redistimeseries_port ${{ secrets.PERFORMANCE_RTS_PORT }}
               --redistimeseries_pass '${{ secrets.PERFORMANCE_RTS_AUTH }}'
+
+      - name: Fail flow if benchmark failed
```

Comment:
unnecessary (without the `continue-on-error: true`, the job will fail if these steps fail)